packages/react/src/use-action-form/types.ts

```diff
@@ -54,6 +54,8 @@ export type UseActionFormResult<
   actionData?: ActionFunc extends ActionFunction<GivenOptions, any, any, SchemaT, any>
     ? ReturnType<typeof useAction<GivenOptions, SchemaT, ActionFunc, any>>[0]["data"]
     : ReturnType<typeof useGlobalAction<any>>[0]["data"];
+
+  originalFormMethods: UseFormReturn<FormVariables, FormContext>;
```

Comment:
Did you end up actually needing this?

Reply:
Yes, this was required in order to get types to work for a <FormProvider> which seems required for our plan for custom components